Gopkg.toml

```diff
@@ -72,7 +72,7 @@ required = [
 
 [[constraint]]
   name = "github.com/openshift/cluster-api-provider-libvirt"
-  revision = "c23986b36a2521cd69ac22d7adb74a22bca9f7aa"
+  branch = "master"
```

Comment:
Shouldn't we put the revision instead of using `master` otherwise if someone add/remove/update the dependency then this will also update which might result to failure?